core/frontend-stubs/cli-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/cli/AntennaCLISettingsReader.java

```diff
@@ -203,4 +209,22 @@ public ToolConfiguration readSettingsToToolConfiguration(XmlSettingsReader reade
 
         return toolConfigBuilder.buildConfiguration();
     }
+
+
+    private Map<String, Object> mapProperties(File propertiesFile) {
+        try (InputStream input = new FileInputStream(propertiesFile)) {
+            Properties prop = new Properties();
+
+            prop.load(input);
+
+            return prop.entrySet()
+                    .stream()
+                    .collect(Collectors.toMap(
```

Comment:
Wasn't there at some point a filter which only allowed uppercase keys?

Reply:
This is reading in the properties file. We did not set any restrictions for those variables.

The uppercase keys you are referring to are in the TemplateRenderer for the system environment variables

Reply:
oh, sure, right.